app/components/Navbar.tsx

```diff
@@ -15,9 +15,7 @@ import { useMemo } from 'react';
 
 import { NetworkButton } from './NetworkButton';
 import { trackEvent } from 'fathom-client';
-import BetsIcon from '@/app/components/ui/icons/BetsIcon';
```

Comment:
If we don't use the `Icon` and `LifiWidgetPopover` components, we could remove those too.

Reply:
We can remove Icon, `LifiWidgetPopover` was removed by the merged now fixed. thanks